src/osvr/Util/LogRegistry.cpp

```diff
@@ -87,22 +87,39 @@ namespace util {
             auto spd_logger = spdlog::get(logger_name);
```

Comment:
@rpavlik It only makes a copy of a `shared_ptr`. If the copy operation throws, then we can't save you anyway (because copying a fallback `shared_ptr` would also likely throw). I figure it's best to let any OOM errors just propagate up the stack.